tools/klee/main.cpp

```diff
@@ -636,13 +634,11 @@ static int initEnv(Module *mainModule) {
   */
 
   Function *mainFn = mainModule->getFunction(EntryPoint);
-  if (!mainFn) {
+  if (!mainFn)
```

Comment:
Is this klee policy to remove brackets for 1 line ifs? I thought keeping them is considered better practice?

Reply:
Good question. I try to orientate myself according to the LLVM coding standard but still trying to resemble the code style used around the KLEE code I'm modifying.

Reply:
Interesting, haven't coke across a style guide that doesn't use braces, but I guess LLVM does ...